### PR TITLE
Add WebApps working group status

### PR DESCRIPTION
**Pull request by sergio-carlavilla**

### CODE REVIEW

File: 2021q4/_index.adoc [hunk 1]
```diff
@@ -29,6 +29,8 @@ toc::[]
 
 Entries from the various official and semi-official teams, as found in the link:../../administration/[Administration Page].
 
+include::{reports-path}/www.adoc[]
+
```
Comment (pauamma) [inline 2021q4/_index.adoc:33]: @debdrup, I think you requested no changes to _index.adoc for your workflow and or sanity?

**Review by pauamma — COMMENTED** [review]
Leaving approval/change request until answer about index, but LGTM otherwise.

File: 2021q4/_index.adoc (hunk 1 above)
Comment (debdrup) [reply 2021q4/_index.adoc:33]: Yes, I'd much prefer if _index.adoc is maintained by the status reports team.

**Review by pauamma — CHANGES_REQUESTED** [review]
Can you revert the 2021q4/_index.adoc changes and keep the rest?

File: 2021q4/_index.adoc (hunk 1 above)
Comment (sergio-carlavilla) [reply 2021q4/_index.adoc]: Sorry, fixed

**Review by pauamma — APPROVED** [review]

**Pull request merged**